app/views/posts/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<section class="comment-container container">
+<%= form_with model: @post, url: user_posts_path , class: 'user-card flex-col' do |f| %>
+<%= f.label :title, "Post title:" %>
+<%= f.text_field :title class: 'input'%>
```

Comment:
- [ ] Kindly fix the syntax error in your new post form. You should add a comma after `:title`.

  ```
  <%= f.text_field :title, class: 'input'%>
  ```

  ![image](https://github.com/Mofuhidy/blog-app/assets/81468675/6ed92144-3dbe-4e92-86ff-aa29d1b8c61a)